### PR TITLE
Add sandbox properties

### PR DESCRIPTION
**Pull request by GZolla**
This allows demo snippets to  be shown in the sandbox pages.

### CODE REVIEW

File: components/table/README.md
[the remark this answers is not in the document]
Comment (GZolla) [reply components/table/README.md:1]: Yes, we would first need to make the list of the demos we would want to add, for now I was just adding anything with a name/component attached to it.

File: components/dropdown/README.md [hunk 1]
```diff
@@ -123,7 +123,7 @@ To make your `d2l-dropdown-button-subtle` accessible, use the following properti
 ## Opener: Context Menu [d2l-dropdown-context-menu]
 `d2l-dropdown-context-menu` is a simple/minimal opener for dropdown content (`d2l-dropdown-content`, `d2l-dropdown-menu` or `d2l-dropdown-tabs`).
 
-<!-- docs: demo code properties name:d2l-dropdown-context-menu align:flex-start autoOpen:true autoSize:false size:medium -->
+<!-- docs: demo code properties name:d2l-dropdown-context-menu sandboxTitle:'Dropdown Context Menu' align:flex-start autoOpen:true autoSize:false size:medium -->
```
Comment (margaree) [inline components/dropdown/README.md:126]: We likely want to be consistent in the titles (e.g., have the openers all have the "Dropdown with X opener")

File: components/filter/README.md [hunk 2]
```diff
@@ -237,7 +237,7 @@ This component is built to be used alongside the [d2l-filter-dimension-set](#d2l
 
 This component is built to be used alongside the [d2l-filter-dimension-set](#d2l-filter-dimension-set) component. It will give you a selectable filter value based on the `range` defined on the component, which is to be one of a set of pre-defined range options. Selection triggers the `d2l-filter-change` event, with `start-value` and `end-value` (in UTC) being included in the changes for the `selected` item.
 
-<!-- docs: demo code properties name:d2l-filter-dimension-set-date-text-value align:start autoOpen:true autoSize:false size:large -->
+<!-- docs: demo code properties name:d2l-filter-dimension-set-date-text-value sandboxTitle:'Dimension Set Value - Preset Date Range' align:start autoOpen:true autoSize:false size:large -->
```
Comment (margaree) [inline components/filter/README.md:240]: We should have "Filter" at the front of all of these

File: components/table/README.md [hunk 3]
```diff
@@ -244,7 +244,7 @@ Note that the example below hides much of the implementation. See the code in [M
 
 This is a radio menu item to be used within the `d2l-table-col-sort-button` component for a [multi-faceted sort](#multi-faceted-sort-button).
 
-<!-- docs: demo code properties autoSize:false align:start name:d2l-table-col-sort-button-item -->
+<!-- docs: demo code properties autoSize:false align:start name:d2l-table-col-sort-button-item sandboxTitle:'Table  Sortable Column Button - Slotted Item' -->
```
Comment (margaree) [inline components/table/README.md:247]:
```suggestion
<!-- docs: demo code properties autoSize:false align:start name:d2l-table-col-sort-button-item sandboxTitle:'Table Sortable Column Button - Slotted Item' -->
```

**Review by margaree — APPROVED** [review]

**Pull request merged**